nautobot/core/views/mixins.py

```diff
@@ -241,8 +241,6 @@ def get_permissions_for_model(self, model, actions):
         """
         model_permissions = []
         for action in actions:
-            if action not in ("view", "add", "change", "delete"):
-                raise ValueError(f"Unsupported action: {action}")
```

Comment:
Removing this part since we are supporting custom actions that might not be in these typical action list.